### PR TITLE
add bureaucrat protect to testwiki

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -276,6 +279,12 @@
 			),
 		),
 	),
+	// Restrictions
+	'+wgRestrictionLevels' => array(
+		'default' => array(
+			'bureaucrat',
+	),
+		),
```
Comment (Southparkfan) [inline LocalSettings.php]: Unneeded line.

File: LocalSettings.php [hunk 2]
```diff
@@ -265,7 +265,10 @@
 			),
 			'wikicreator' => array(
 				'createwiki' => true,
-			),
+		'+testwiki' => array(
+			'bureaucrat' => array(
+				'bureaucratprotect' => true,
+		
 		),
```
Comment (Southparkfan) [inline LocalSettings.php]: This closed the "bureaucrat => array", not the "+testwiki => array" array.

**Comment by Reception123** [comment]
@Southparkfan So we don't need WgRestrictionLevels?

**Comment by Reception123** [comment]
Now #31

**Pull request closed without merging**